spec/models/user_spec.rb

```diff
@@ -118,39 +98,17 @@
         user.reload
         user.mobile = num
         user.save
-        expect(user.errors.any?).to be_falsey
+        expect(user.errors.any?).to be_falsey, user.errors.messages.inspect
```

Comment:
I meant both, but for now, we can just remove the addition of the inspect, which I believe is unnecessary, right?